website/src/site/markdown/mailing-list.md

```diff
@@ -16,22 +16,31 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->
-#Mailing List
+# Mailing List
 
+- [REEF User](#reef-user)
 - [REEF Dev](#reef-dev)
 - [Commits](#commits)
 
 **Note:** These email addresses are automated. Just writing "Subscribe" or "Unsubscribe" in both the subject line and the body will suffice and a confirmation email with further instructions will be sent as a reply to your email after some delay.
 
-###<a name="reef-dev"></a>REEF Dev Mailing List
+### <a name="reef-user"></a>REEF User Mailing List
+
+This is the list to ask questions about and discuss issues with using REEF.
+
+- [Subscribe](mailto:user-subscribe@reef.apache.org) (user-subscribe@reef.apache.org)
+- [Unsubscribe](mailto:user-unsubscribe@reef.apache.org) (user-unsubscribe@reef.apache.org)
+- [Archive](http://mail-archives.apache.org/mod_mbox/reef-user/)
```

Comment:
It is expected and should change a couple of hours after the first emails are sent to the list